openthread/Kconfig

```diff
@@ -4,14 +4,16 @@
 # SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
 #
 
+if NET_L2_OPENTHREAD
```

Comment:
Nothing wrong, nut I would like to understand the difference between this Kconfig file, and this:
https://github.com/nrfconnect/sdk-nrf/blob/master/subsys/net/openthread/Kconfig

What goes where, and why ?